webapp/src/types/types.ts

```diff
@@ -564,6 +564,7 @@ export interface TaskView {
   date: string;
   type: TaskTypeValue | "DOWNLOAD" | "LAUNCH" | "UNKNOWN";
   status: string;
+  userName: string;
```

Comment:
`userName?: string;`

then

`<TableCell align="right">{row.userName || ""}</TableCell>`

Allows to have the default value in one place.

Reply:
Can you explain why?